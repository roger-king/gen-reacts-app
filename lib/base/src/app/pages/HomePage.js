"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var React = require("react");
var HomePage = function (props) {
    return <h1> Home Page </h1>;
};
exports.default = HomePage;
//# sourceMappingURL=HomePage.js.map