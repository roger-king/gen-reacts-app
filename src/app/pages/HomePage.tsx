import * as React from 'react';

const HomePage: React.SFC = props => {
  return <h1> Home Page </h1>;
};

export default HomePage;
