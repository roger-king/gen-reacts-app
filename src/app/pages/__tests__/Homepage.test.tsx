import * as React from 'react';
import { create } from 'react-test-renderer';
import HomePage from './../HomePage';

test('Snapshot of Home Page', () => {
    const page = create(<HomePage />).toJSON();
    expect(page).toMatchSnapshot();
});
