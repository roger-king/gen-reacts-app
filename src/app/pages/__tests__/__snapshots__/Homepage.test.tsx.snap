// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Home Page 1`] = `
<h1>
   Home Page 
</h1>
`;
